### PR TITLE
Fixed homepage appearance in mobile view

### PR DESCRIPTION
**Pull request by nancy2681**
Fixed issue #5797 

Mobile View

Before :

image

After:

image

Web View

Before:

image

After:

image

### CODE REVIEW

**Comment by nancy2681** [comment]
@kyletsang Can you please review!?

File: www/src/pages/index.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ const ButtonToolbar = styled('div')`
     margin: -1rem;
 
     & > * {
-      width: 100%;
+      width: 90%;
```
Comment (kyletsang) [inline www/src/pages/index.js:87]: I think it's better to use `calc()` here because the width does not account for the margins hence the overflow

**Comment by thelokeshgoel00** [comment]
The Website doesn't looks good after the update on desktop, this reduced height isn't better. Maybe, separate code base for height on mobile and desktop

**Comment by nancy2681** [comment]
> The Website doesn't looks good after the update on desktop, this reduced height isn't better. Maybe, separate code base for height on mobile and desktop

There is no significant visible difference I can see with this code change. If you can send screenshot we will get to know more, otherwise i think everything is working fine

**Comment by nancy2681** [comment]
After using calc():

image

**Comment by nancy2681** [comment]
@kyletsang I have made required changes. Please review

File: www/src/pages/index.js [hunk 2]
```diff
@@ -84,7 +84,7 @@ const ButtonToolbar = styled('div')`
     margin: -1rem;
 
     & > * {
-      width: 100%;
+      width: calc(100% - 1.85rem);
```
Comment (kyletsang) [inline www/src/pages/index.js:87]: Should actually be `calc(100% - 2rem)` since the margins added on each side sum up to 2.

**Comment by nancy2681** [comment]
@kyletsang Made the required changes. Can you please review!

**Review by jquense — APPROVED** [review]

**Review by kyletsang — APPROVED** [review]

**Pull request merged**